backend/src/controllers/clients.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Response } from 'express';
+import { AuthRequest } from 'common/auth.middleware';
+import { IClient } from "shared-types";
+import { BaseController } from "./base.controller";
+import clientModel from "../models/client.model";
+import httpStatus from 'http-status';
+import { isEmpty } from 'lodash';
+import { signTokens } from '../utils/tokens';
+
+class ClientsController extends BaseController<IClient> {
+  constructor() {
+    super(clientModel);
+  }
+
+  async updateByBusinessId(req: AuthRequest, res: Response){
+    const fields = ['businessName', 'businessDescription'];
+    const { businessId } = req.body;
```

Comment:
You don't have to send the `businessId` in the body.
You have it in the `req.user` object